components/dashboard/src/App.tsx

```diff
@@ -57,6 +58,11 @@ const App: FC = () => {
         return <AppLoading />;
     }
 
+    // Page can be loaded even if user is not authenticated
```

Comment:
This does not go into the regular app router because all of those routes account for authenticated pages.